evalml/automl/automl_search.py

```diff
@@ -383,18 +385,24 @@ def search(self, X, y, data_checks="auto", feature_types=None, show_iteration_pl
             except NameError:
                 show_iteration_plot = False
 
-        # make everything pandas objects
-        if not isinstance(X, pd.DataFrame):
-            X = pd.DataFrame(X)
+        # make everything ww objects
+        if not isinstance(X, ww.DataTable):
```

Comment:
Yeah... I was hesitant to add this in now since everywhere will blow up (our tests) with this warning. But I'll add it for now just as a logger warning and not a python warning.

Reply:
Just added (and added `test_input_not_woodwork_logs_warning` as a test :d)